default.nix

```diff
@@ -21,8 +21,16 @@
 # Once the build is finished, you will find, in the current directory,
 # a symlink to where Coq was installed.
 
-{ pkgs ? (import <nixpkgs> {}), ocamlPackages ? pkgs.ocamlPackages,
-  buildIde ? true, buildDoc ? true, doCheck ? true }:
+{ pkgs ?
+    (import ((import <nixpkgs> {}).fetchzip {
```

Comment:
Simple curiosity: why `fetchzip` rather than `fetchFromGitHub`? Is it more efficient?

Reply:
Because `fetchFromGitHub` is noise around `fetchzip`.

Reply:
Oh OK, well it avoids repeating the entire archive URL, but well to each their own.

Reply:
You could use the `fetchTarball` builtin directly, couldn't you?